packages/recommend/src/__tests__/getRecommendations.test.ts

```diff
@@ -0,0 +1,84 @@
+import { TestSuite } from '../../../client-common/src/__tests__/TestSuite';
+
+const recommend = new TestSuite('recommend').recommend;
+
+function createMockedClient() {
+  const client = recommend('appId', 'apiKey');
+  // @ts-ignore read-only property
+  client.transporter = {
+    ...client.transporter,
+    read: jest.fn(),
```

Comment:
can this be done with jest.spyOn instead to avoid the mutation? not a big deal though

Reply:
Good point—see 9d919a63dbb71d38486b3eab7ff2c724a7de36c3.